data/mods/Xedra_Evolved/mutations/classes.json

```diff
@@ -4,7 +4,7 @@
     "id": "DREAMER",
     "name": "Dreamer",
     "points": 0,
-    "description": "Dreamers are people able to tap into the power of dreams and nearby dimensions and create fantastical effects as a result.  They can even pull some of this dream material into our world to build other things with it.  Gains a spell to summon dreamdross scraps on demand.",
+    "description": "Dreamers are people able to tap into the power of dreams and nearby dimensions and create fantastical effects as a result. They can even pull some of this dream material into our world to build other things with it. Gains a spell to summon dreamdross scraps on demand",
```

Comment:
```suggestion
    "description": "Dreamers are people able to tap into the power of dreams and nearby dimensions and create fantastical effects as a result.  They can even pull some of this dream material into our world to build other things with it.  Gains a spell to summon dreamdross scraps on demand",
```

Reply:
Is removing the trailing period intended?